django_declarative_apis/machinery/errors.py

```diff
@@ -113,7 +112,7 @@ def __init__(self, additional_info=None):
             http.client.responses.get(http.client.NOT_FOUND),
         )
         if additional_info:
-            error_message += " : " + additional_info
```

Comment:
This change (removing a space from the message to make it consistent with the other messages) requires a small test fix in VE.